apps/change-log.md

```diff
@@ -16,6 +16,16 @@ Of course, we will continue to inform you if there are any breaking changes to t
 
 ## 2023-05-11
 
+### Changes with software release 7.84.0
+
+#### <i class="fas fa-plus"></i> New
+
+* [`PATCH`/{shopId}/products/{productId}/custom-attributes](https://developer.epages.com/apps/api-reference/patch-shopid-products-productid-custom-attributes.html)
+
+<hr>
+
+## 2023-05-11
```

Comment:
The date twice? Same as in line 17.

Reply:
@AliceStatler Uh, nice catch. That's wrong, of course, and needs to be ## 2023-06-08